### ISSUE TITLE
Not inspecting modules when doing so is enabled

### ISSUE THREAD

**Issue by BenMcClainCarpe** [terraform-linters/tflint-ruleset-aws#120]
# Problem
AWS linting is not looking into modules when the `--module` argument is included.  Vanilla tflint rules are checking the modules.

Have the following files:
```
.
├── main.tf
└── module
    └── main.tf
```

Contents of `./main.tf`:
```tf
provider "aws" {
  region = "us-west-2"
}


module "derp" {
  source = "./module"
}
```

Contents of `module/main.tf`
```tf
resource "aws_instance" "derp" {
  ami = "ami-ssjjdisosos"
  instance_type = "t6.micro"
}
```

Command run:
```sh
tflint . --module
```

## Expected Outcome
Throw an error due to the wrong `instance_type`.

## Current Outcome
No errors or warnings found.

## Additional notes:
If I move the `aws_instance` resource into the root main.tf it throws the error as expected.

```sh
tflint -v
TFLint version 0.28.1
+ ruleset.aws (0.4.0)
```

**Comment by BenMcClainCarpe** [comment]
I was mistaken about the non-AWS tflint rules checking within the module.  At least we hav consistency now.  This does not appear to be an issue with tflint-ruleset-aws.  Sorry if I wasted anyone's time.
